### PR TITLE
CORS headers

### PR DESCRIPTION
**Pull request by PyDev777**

### CODE REVIEW

File: data_converter/settings.py [hunk 1]
```diff
@@ -61,13 +63,55 @@
...
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# CORS settings
+# https://github.com/adamchainz/django-cors-headers
+
+CORS_ORIGIN_WHITELIST = [
+    'https://dataocean.us',
+    'http://127.0.0.1:8000',
```
Comment (roman-tiukh) [inline data_converter/settings.py:79]: move this to settings_local and add localhost for developing to test server
```suggestion
    'http://127.0.0.1:8000',
    'http://127.0.0.1:3000',
    'http://localhost:3000',
```

File: data_converter/settings.py [hunk 2]
```diff
@@ -61,13 +63,55 @@
...
+# CORS_URLS_REGEX = r'^/api/.*$'
+
+CORS_ALLOW_METHODS = [
+    'DELETE',
+    'GET',
+    'OPTIONS',
+    'PATCH',
+    'POST',
+    'PUT',
+]
```
Comment (roman-tiukh) [inline data_converter/settings.py:95]: that is default value
```suggestion
```

File: data_converter/settings.py [hunk 3]
```diff
@@ -61,13 +63,55 @@
...
+# https://github.com/adamchainz/django-cors-headers
+
+CORS_ORIGIN_WHITELIST = [
+    'https://dataocean.us',
+    'http://127.0.0.1:8000',
+]
+
+CSRF_TRUSTED_ORIGINS = [
+    'dataocean.us',
+]
```
Comment (roman-tiukh) [inline data_converter/settings.py:84]: why we need this?
```suggestion
```

File: data_converter/settings.py [hunk 4]
```diff
@@ -61,13 +63,55 @@
...
+    'accept',
+    'accept-encoding',
+    'authorization',
+    'content-type',
+    'dnt',
+    'origin',
+    'user-agent',
+    'x-csrftoken',
+    'x-requested-with',
+]
```
Comment (roman-tiukh) [inline data_converter/settings.py:107]: that is default value
```suggestion
```

**Comment by criticus** [comment]
@roman-tiukh please do not merge with no reviews

**Pull request merged**